engine/gamesys/src/gamesys/scripts/script_http_js.cpp

```diff
@@ -214,6 +221,7 @@ namespace dmGameSystem
             ctx->m_Callback = callback;
             ctx->m_Requester = sender;
             ctx->m_RequestData = request_params.m_SendData;
+            ctx->m_Path = path;
```

Comment:
I got a bunch of issues using `path`: 

>"DEFOLD:ERROR:RESOURCE: Could not resolve a mount path '/dat+'\n"

>"DEFOLD:ERROR:RESOURCE: Could not resolve a mount path '�,�'\n"

>"DEFOLD:ERROR:RESOURCE: Could not resolve a mount path 'x7�'\n"

I take a look PR again, and see that we don't copy string to manage it ourself, so it's Lua that manages it.  I think we should memcopy this string as ell as we do for headers and free it when we don't need it anymore